tests/unity/auto/generate_config.yml

```diff
@@ -18,7 +18,7 @@
       - Defs.h
       - Board.h
       - Exception.h
-  :boilerplates: 
+  :boilerplates:
```

Comment:
Same as with json-patch-tests. The upstream for this is https://github.com/ThrowTheSwitch/Unity